metadata-events/mxe-utils-avro/src/main/resources/v1/avro/com/linkedin/mxe/MetadataChangeLog.avsc

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
Does this mean we're basically making snapshots of the generated avro schemas and storing them as versions whenever we make changes along with the latest?